pkg/ot/metrics.go

```diff
@@ -20,33 +20,20 @@ import (
 	crprometheus "github.com/prometheus/client_golang/prometheus"
 	"go.opentelemetry.io/otel/exporters/prometheus"
 	"go.opentelemetry.io/otel/metric/global"
-	"go.opentelemetry.io/otel/sdk/metric/aggregator/histogram"
-	controller "go.opentelemetry.io/otel/sdk/metric/controller/basic"
-	"go.opentelemetry.io/otel/sdk/metric/export/aggregation"
-	processor "go.opentelemetry.io/otel/sdk/metric/processor/basic"
-	selector "go.opentelemetry.io/otel/sdk/metric/selector/simple"
+	"go.opentelemetry.io/otel/sdk/metric"
 	"sigs.k8s.io/controller-runtime/pkg/metrics"
 )
 
 // RegisterMetrics enables prometheus metrics for OpenTelemetry.
 func RegisterMetrics() error {
-	config := prometheus.Config{
-		Registry: metrics.Registry.(*crprometheus.Registry), // use the controller runtime metrics registry / gatherer
-	}
-	c := controller.New(
-		processor.NewFactory(
-			selector.NewWithHistogramDistribution(
```

Comment:
All of this aggregation and temporality configuration turns out to be the default in the newly refactored `sdk/metrics` otel package, so we can drop it.